index.html

```diff
@@ -4769,6 +4769,11 @@ <h3>User Agent Processing</h3>
 				<p>If a table of contents element is not found, the publication does not have a table of contents that
 					can be used for machine rendering purposes.</p>
 
+				<p class="note">This algorithm is not defined in purely event driven terms, as inspecting all descendant
+					nodes is not necessary to obtain the needed information from the DOM. In some cases, an element is
+					exited immediately after it is processed. An event approach could be applied, but would require
+					modifying the algorithm to process/ignore the omitted nodes.</p>
+
 				<ol>
```

Comment:
(1) I think this note should come after the first paragraph. Otherwise, I believe the text is o.k.

(2) The usual terminology in the algorithm is 

> exit the element and continue processing with the next element.

Maybe a slightly better way is to emphasize the situation, something like

> exit the element without descending the sub-tree at this element and continue processing with the next element.

A bit more verbose but may help making things clearer...

Reply:
This looks like it's good now, too. The only thing I did was s/omitted/skipped/ in the note to match the terminology from the other PR.